config/software/mingw.rb

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2012-2015 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "mingw"
+default_version "5.1.0-tdm64-1"
+
+dependency "msys-base"
+dependency "binutils"
+dependency "mingw-runtime"
```

Comment:
What is the purpose of the separated `msys-base` and `mingw-runtime` software defs? Will any other software definitions use these besides `mingw`? It feels like we would want this bundle of software versioned as a unit and any other dependent software defs would just depend on `mingw`.